masterlist.yaml

```
@@ -14987,6 +15003,28 @@ plugins:
         util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
 
+  - name: '(HandtoHand(Armor|Locksmith)?|H2HBooksAndTrainers)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/59790/'
+        name: 'Hand to Hand - An Adamant Addon'
+  - name: 'HandtoHand.esp'
+    tag:
+      - Names
+      - Text
+  - name: 'HandtoHandArmor.esp'
+    tag: [ Text ]
+  - name: 'HandtoHandLocksmith.esp'
```

One last stylistic nitpick that I missed initially. The plugin as downloaded & installed is capitalized differently, so we should match that:
```suggestion
  - name: 'HandToHandLocksmith.esp'
```